doc/bdDijkstra/pgr_bdDijkstraCost.rst

```diff
@@ -7,6 +7,17 @@
     Alike 3.0 License: https://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/pgr_dijkstraCost.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_dijkstraCost.html>`__)
```

Comment:
The html file names should match the one on 3.1

Reply:
The file name is `pgr_bdDijkstraCost.rst` So after the `/en/` it should read `pgr_bdDijkstraCost.html`